Sources/Core/ModernAVPlayer.swift

```diff
@@ -102,21 +102,45 @@ public final class ModernAVPlayer: NSObject, ModernAVPlayerExposable {
     ///
     /// Sets the media current time to the specified position
     ///
-    /// - Note: position is bounded between 0 and end time or available ranges
+    /// - Note: position is bounded between 0 and end time or available ranges, resulting seek position may differ slightly for efficiency
     /// - parameter position: time to seek
     ///
     public func seek(position: Double) {
```

Comment:
don't think this method is useful anymore. 
default value on new associated method you created should be enough and  backward compatible

Reply:
I agree that would be better, but if the user had an interface to abstract from ModernAVPlayer removing seek(position: Double) will cause their code to break, maybe it is good to mark that func as **deprecated** instead and leave for a while

Reply:
ok for deprecated message, thanks